src/ssh.js

```diff
@@ -248,8 +248,12 @@ class SSHConnector {
         return contains === expect;
     }
 
-    checkVirt() {
-        return 'TODO'; // TODO: add shell command for checking virtualization.
+    async checkVirt() {
+        if(await this.exec(context, 'cat /proc/cpuinfo | grep -E -c 'svm|vmx'') == 0){
+		if(await this.exec(context, 'cd /dev/ | ls | grep -c 'kvm'') == 0){
```

Comment:
I'm also not sure about the order these two _if conditions_ are meant to run (because of missing `}`). So I will be able to review better after that is fixed.